.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,48 @@
+name: Benchmarks
+
+on: [pull_request]
+
+jobs:
+  benchmarks:
+    name: benchmarks
+
+    # We can't currently run benchmarks on PRs from forked repos, because the
+    # tachometer action reports results by posting a comment, and we can't post
+    # comments without a github token.
+    if: github.event.pull_request == null || github.event.pull_request.head.repo.full_name == github.repository
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Restore node_modules
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          # Bump the number after `lit-1.x-node-modules` to force a cache
+          # update. Note there are multiple cache actions in `tests.yml` that
+          # should all keep keys in sync.
+          key: lit-1.x-node-modules-0-${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+
+      - name: NPM install
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Benchmark
+        run: npx tach --config tachometer.json --json-file results.json
+
+      - name: Report
+        uses: andrewiggins/tachometer-reporter-action@v2
+        with:
+          report-id: lit-html-1.x
```

Comment:
I don't know what this field does. Should I remove it?

Reply:
@aomarks ?

Reply:
I'm going to try out a quick branch without this to see what happens.

Reply:
Oh, just stumbled on this commit: https://github.com/andrewiggins/tachometer-reporter-action/commit/0f505a75d1451d2c19645731de1bfb6ce9e7d0dc

It's been removed from the docs, so I'm assuming it's not supported anymore. Also, nothing in my test branch (https://github.com/lit/lit/pull/1975) seems to have changed other than that the initial comment stating that the benchmarks were in progress didn't appear and the section title in the results comment changed from "lit-html-1.x" to "this-change, lit-html-1.x, previous-1.x-release".